pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   meta: ^1.1.0
   path: ^1.6.0
   protobuf: ^2.0.0
-  sass: ^1.42.0
+  sass: 1.49.0
```

Comment:
Is `^` intentionally removed here?

Reply:
Yes, now that we're releasing in lockstep we want to ensure that each version of the embedded compiler uses precisely the same version of Dart Sass.